content/induction/sprints/1/prep/index.md

```diff
@@ -0,0 +1,353 @@
++++
+title = 'prep'
+description = 'You can use these notes to prepare for the first in-person session of the software development course'
+layout = 'prep'
+emoji= '📝'
+menu_level = ['sprint']
+weight = 1
+backlog= 'Module-Induction'
+backlog_filter= 'Induction'
++++
+
+## Learning objectives
```

Comment:
Can you list these in the front matter so they can be picked up by other views, please?

Same setup as a block/index.md